tests/tests_unit/test_graph/test_loaders/test_rdf_to_instance_space.py

```diff
@@ -0,0 +1,79 @@
+from collections.abc import Iterable
+
+import pytest
+from cognite.client.data_classes import AssetWriteList
+from rdflib import RDF, Literal, Namespace
+
+from cognite.neat.core._client.testing import monkeypatch_neat_client
+from cognite.neat.core._constants import DEFAULT_SPACE_URI
+from cognite.neat.core._instances.extractors import AssetsExtractor, BaseExtractor
+from cognite.neat.core._instances.loaders import InstanceSpaceLoader
+from cognite.neat.core._shared import Triple
+from cognite.neat.core._store import NeatInstanceStore
+from cognite.neat.core._utils.upload import UploadResult
+from tests.data import InstanceData
+
+
+def load_instance_spaces_test_cases():
```

Comment:
Can you add a test case to cover when graph_store is None? What is the exception/logging/NeatIssue we are sending to the user/session when the whole thing is None.